apps/astarte_pairing_api/lib/astarte_pairing_api_web/router.ex

```diff
@@ -18,10 +18,12 @@
 
 defmodule Astarte.Pairing.APIWeb.Router do
   use Astarte.Pairing.APIWeb, :router
+  alias Astarte.Pairing.APIWeb.Plug.LogRealm
 
   pipeline :realm_api do
     plug :accepts, ["json"]
-    plug Astarte.Pairing.APIWeb.Plug.LogRealm
+    plug LogRealm
+    plug Astarte.Pairing.APIWeb.Plug.AuthorizePath
```

Comment:
This is wrong: in the `/devices` scope, the authorization header should contain the Credentials Secret of the device, not an Astarte JWT. See https://docs.astarte-platform.org/astarte/latest/050-pairing_mechanism.html#pairing-facilities